03/app.js

```diff
@@ -5,14 +5,54 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
-
-
-
-
-
-
-
 function randomNumber(min, max) {
-    return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+	return Math.round(Math.random() * (max - min) + min);
+}
+
+const getSum = (a, b, c) => {
+	const numA = parseInt(a);
+	const numB = parseInt(b);
+	const numC = parseInt(c);
+
+	const intArray = [numA, numB, numC];
+	intArray
+		.sort((a, b) => {
+			return b - a;
+		})
+		.pop();
+
+	const sumFromArray = intArray.reduce((previousVal, currentVal) => previousVal + currentVal, 0);
+	return sumFromArray;
+};
+const sum = getSum(a, b, c);
+
+const isEven = function (number) {
+	if (!Number.isSafeInteger(number)) return null;
+	// if (typeof number === 'number') return null;
+
+	const oddOrEven = number % 2;
+
+	if (oddOrEven === 0) return true;
+	else return false;
```

Comment:
Jesli mamy coś takeigo to zdecydowanie lepiej zrobić `return oddOrEven === 0` lub  `return !oddOrEven`